content/blog/automate-boot-from-san-target-configurations-using-redfish.md

```diff
@@ -0,0 +1,277 @@
+
+Download and install the [HPE RESTful Interface Tool](https://www.hpe.com/info/resttool)  (version 4.1.0.0 or later) on the system you will initiate the automation actions from.
+Once the RESTful Interface tool (referred to as iLOrest in the remainder of the document), you can review the current boot target and their order of execution by following the steps below
+
+`ilorest login <ilo-ip> -u <ilo-user> -p password`
+
+`ilorest bootorder`
+
+Refer to the [iLOrest documentation](https://hewlettpackard.github.io/python-redfish-utility/#bootorder-command) for more information on this command.
+
+# Configuration Steps
+
+Now that you have gathered the pre-requisite information and installed the ilorest tool, let’s discuss the detailed steps for implementing the automation.
+
+## Step 1: Ensure that the FCScanPolicy value is set to ‘AllTargets’
+
+`ilorest  get FCScanPolicy --selector Bios`
+
+Expected output: FCScanPolicy=AllTargets
+
+If the value is not set to AllTargets, you can change it using the following:
+
+`ilorest  set "FCScanPolicy"="AllTargets" --selector Bios`
+
+`ilorest commit`
+
+`ilorest reboot`
+
+After rebooting the server, you can verify that the settings have taken effect using the get FCScanPolicy command or navigating to the BIOS menus as shown in Figure 1.
+
+![](/img/picture1.png "Figure 1: FCScanPolicy configuration")
+
+Step 2: Identify the right FC Card
+The server you are trying to configure might have one or more FC cards in it, and you need to identify the right card using the steps below.
+
+List out available network adapters (includes FC cards)
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/`
+
+The previous command returns the following output: 
+
+`{
+  "@odata.context": "/redfish/v1/$metadata#NetworkAdapterCollection.NetworkAdapterCollection",
+  "@odata.etag": "W/"F09DE05C"",
+  "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/",
+  "@odata.type": "#NetworkAdapterCollection.NetworkAdapterCollection",
+  "Description": "The collection of network adapter resource instances available in this chassis.",`
+
+`"Members": [
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DC080000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE083000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE081000/"
+    },
+    {
+      "@odata.id": "/redfish/v1/Chassis/1/NetworkAdapters/DE07B000/"
+    }
+  ],`
+
+`"Members@odata.count": 4,
+…`
+
+Make a note of the device IDs highlighted in the output above.  The values for these highlighted items might be different on the server you are using.
+
+Query these items using the device ID to verify it is the right FC card that you would like to configure, for example using DE083000 device (your output and device IDs might be different).
+
+`ilorest rawget /redfish/v1/Chassis/1/NetworkAdapters/DE083000`
+
+Look for the following in the output.
+
+`"Location": {
```

Comment:
Same remark as above. Please use a code block snippet for the output of the command.